cypress.json

```diff
@@ -1,4 +1,4 @@
 {
     "baseUrl": "http://0.0.0.0:9990",
-    "video": false
+    "video": true
```

Comment:
Should this still be enabled?

Reply:
Yeah I use it a lot for debugging and I don't notice a speed difference when running the tests so it's usually just a pain to keep changing this setting